android/library/src/main/java/io/logto/android/Logto.kt

```diff
@@ -50,12 +51,36 @@ object Logto {
                     onComplete(error, null)
                 }
             }
-        ).startAuth(context)
+        ).login(context)
+    }
+
+    fun logoutWithBrowser(
+        context: Context,
+        onComplete: (error: Error?) -> Unit
+    ) {
+        checkInitState()
+        credential?.let {
+            BrowserLogoutFlow.init(
+                logtoConfig,
+                it.idToken,
+            ) {
+                error ->
+                if (error == null) {
+                    clearCredential()
+                }
+                onComplete(error)
+            }.logout(context)
+        }
```

Comment:
maybe no need for `BrowserLoginFlow` and `BrowserLoginFlow`? just `io.logto.android.auth.browser.BrowserFlow` with login/out method will be good.

p.s. we're using sign in / out globally instead of login/out, added [LOG-141](https://linear.app/silverhand/issue/LOG-141/align-engineering-phrases) for detailed alignment.